recipes/sdsl-lite/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  2.1.1:
+    url: https://github.com/simongog/sdsl-lite/archive/refs/tags/v2.1.1.zip
```

Comment:
Is there a reason you're using the auto-generated `v2.1.1.zip` instead of the the offline install tarball?
See https://github.com/simongog/sdsl-lite/releases/tag/v2.1.1